mitodl/ocw-studio
# package version bump

#### Pre-Flight checklist



#### What are the relevant tickets?

none

#### What's this PR do?

just bumps a few package versions, I just ran `yarn upgrade`.

#### How should this be manually tested?

check out the site and make sure all looks good. the build should be passing as well.
